### ISSUE TITLE
Bert vs Electra performances

### ISSUE THREAD

**Issue by pretidav** [google-research/electra#42]
Hello, 

I am trying to compare the time performances of Bert and Electra pre-training. 
Looking at the paper it seems that at fixed FLOPs (i.e. fixed time) electra performances are the best in downstreaming finetuning tasks. 

I am struggling a bit with the comparison since the pre-training tasks are pretty different between the models and then I am not sure they can be directly compared by keeping the same hyper-parameters.  
 
For instance: 
max_seq_len used by Bert seems to stand for the total length of the couple of sentences while 
for Electra is just one sentence, 
this would suggest to half the length for Electra (or double for Bert), am I right? 
On the other hand, if I do so, then the number of masks (or replacement in Electra) would be smaller since it scales as masked_lm_prob*max_seq_len. 
Are there other parameters I should keep into account for this comparison? 
 
I am keeping the number of steps to be the same all the time so I would expect an increased performance in finetuning tasks after electra pretraining. 
  
As usual, any comment or help is really appreciated!

**Comment by clarkkev** [comment]
Hi! There can be multiple sentences in a `max_seq_len` example for ELECTRA as well. If it helps, I added the script we used for computing pre-training FLOPs [here](https://github.com/google-research/electra/blob/master/flops_computation.py).
